Assets/_Core/ExoScriptableObject/Scripts/CardObjectInstanciator.cs

```diff
@@ -0,0 +1,49 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using Random = UnityEngine.Random;
+
+public class CardObjectInstanciator : MonoBehaviour
+{
+    private List<string> _cardInInventoryList = new List<string>();
+    
+    private void Start()
+    {
+        
+        for (int i = 0; i < 7; i++)
+        {
+            string randomCard = CreateRandomCard();
+            bool cardInInventory = false;
+            foreach (string card in _cardInInventoryList)
```

Comment:
Le `foreach` pourrait être remplacé par un appel à la fonction `Array.Find()`.
Et le `for` au dessus, dans le même temps, pourrait être remplacé par un `while` en testant la longueur du tableau `_cardInInventoryList` qui est inférieur à 7